test/gtest/ucp/test_ucp_proto.cc

```diff
@@ -353,3 +358,170 @@ UCS_TEST_P(test_perf_node, replace_null)
 }
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_perf_node, all, "all")
+
+
+class test_rndv_abort : public test_ucp_proto {
```

Comment:
maybe move this test to `test_ucp_peer_failure.cc`?

Reply:
Done.